### ISSUE TITLE
need recommendation for note place="unspecified" 

### ISSUE THREAD

**Issue by balmas** [PerseusDL/tei-conversion-tools#36]
@lcerrato and @simonastoyanova  what is your recommendation for the default value of place in the note elements as we convert them from P4? 

"unspecified" I think is the default right now and default TEI stylesheets for transforming from XML to HTML don't like that.

**Comment by lcerrato** [comment]
For things like Loeb texts, it's going to be "bottom." If I recall, it was usually recorded when something was not at the bottom of the page (e.g. margins).
Some texts will run into bottom opposite, but I don't think we can capture that now without reviewing every note.

**Comment by simonastoyanova** [comment]
Yeah, I 'm not sure we can do a one-fits-all thing here. I was never sure
what these attributes on note actually meant in P4. I guess we could look
at each collection/volume/book to get the structure of things in a bigger
scale than each individual note... What is this attribute used for in
Perseus, though? Does it affect the display? Because it doesn't link the
note to the text or vice versa, so if we don't use it for anything I say
just scrap it. If we do, then I suggest we have a look at the
work/book/whatever it is and change it by work. Better suggestions welcome!

On 16 September 2015 at 16:36, Lisa Cerrato notifications@github.com
wrote:

> For things like Loeb texts, it's going to be "bottom." If I recall, it was
> usually recorded when something was not at the bottom of the page (e.g.
> margins).
> Some texts will run into bottom opposite, but I don't think we can capture
> that now without reviewing every note.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/PerseusDL/tei-conversion-tools/issues/36#issuecomment-140760895
> .

## 

Simona Stoyanova
Research Associate

Digital Humanities
Department of Computer Science
University of Leipzig
Augustusplatz 10
04109 Leipzig, Germany

**Comment by lcerrato** [comment]
I believe that it ties in with the CSS in P4 to the extent that it controls where a note appears.
see http://www.perseus.tufts.edu/hopper/text?doc=Perseus:text:1999.04.0077:chapter=20
